project/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,15 @@
 <!-- THE PR TEMPLATE IS NOT AN OPTION. DO NOT DELETE IT! -->
 
 <!--
+    Show us you choose the right branch.
+    Different branches are used for different things :
+    - 3.x is for everything backwards compatible, like patches and deprecation notices
```

Comment:
Please don't use twig for md file. We can't see the preview. Just see how
the README.md file works. :+1:

Sullivan SENECHAL
Le 12 juin 2016 15:01, "Grégoire Paris" <notifications@github.com> a écrit :

> In project/.github/PULL_REQUEST_TEMPLATE.md
> <https://github.com/sonata-project/dev-kit/pull/155#discussion_r66720565>:
>
> > @@ -1,6 +1,15 @@
> >  <!-- THE PR TEMPLATE IS NOT AN OPTION. DO NOT DELETE IT! -->
> >
> >  <!--
> > +    Show us you choose the right branch.
> > +    Different branches are used for different things :
> > +    - 3.x is for everything backwards compatible, like patches and deprecation notices
>
> I rename the file with a .twig extension then.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/sonata-project/dev-kit/pull/155/files/6120477f42d3a03d07100ffb2edd7276f6c8cb14#r66720565>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABnqNVxYLdlBEo2_J0qSX7gz7Ml6qOQ3ks5qLAMagaJpZM4Iyvs5>
> .
>